src/components/Home.tsx

```diff
@@ -75,7 +78,22 @@ export default function Home({ organizations, selectedOrganization, setSelectedO
     const populateUser = async () => {
       const response = await getUser();
       if (response.requestSucceeded && !cancel) {
-        setUser(response.user ?? undefined);
+        if (response.user) {
+          setUser(response.user);
+          if (!response.user.timeZone) {
+            const browserTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+            const updateResponse = await updateUserProfile({ ...response.user, timeZone: browserTimeZone });
```

Comment:
the only tricky part here is that the timezone needs to be among the support timezones returned by the BE.
We can keep this as-is for now but add a comment to check against BE timezones.

Reply:
Oh, ok. I thought the IANA database was only one

Reply:
The BE will provide the map of time zones like so:
{
  <time zone id>: <locale display name>
}
So 'America/Los_Angeles' can be displayed in user's preferred locale.
If the browser time zone is not in the BE provided mapping, we will use UTC.
We decided to use this approach since browsers have different supported lists.. we picked the common ones across browsers.
Since localization provider will be a layer below user provider, we can move this code into the localization provider once we have it.
My planning hasn't been too good so far..
the code is good as-is, we just need to move it later and add one extra check.